lxd/storage/drivers/driver_dir.go

```diff
@@ -249,16 +249,11 @@ func (d *dir) CreateVolume(vol Volume, filler func(mountPath, rootBlockPath stri
 			return err
 		}
 
-		if shared.PathExists(rootBlockPath) {
```

Comment:
@stgraber @monstermunchkin we still need this bit dont we? Otherwise the final image for the VM won't be the specified size for the VM, it'll be the size of the download volume.

Reply:
@monstermunchkin also can you tweak the comment on line 240 of this file:

"// We expect the filler function to have copied the qcow2 image to the rootBlockPath."

to 

"// We expect the filler function to have converted the qcow2 image to raw into the rootBlockPath."